### PR TITLE
 #157185125 UI Fixes

### PR DESCRIPTION
**Pull request by ghost**
#### What does this PR do?
Fix user interface glitches.
#### How should this be manually tested?
Navigate to the following pages:
- Login
- Dashboard
#### What are the relevant pivotal tracker stories?
[#157185125](https://www.pivotaltracker.com/story/show/157185125)
#### Screenshots (if appropriate)
**Before**
image

**After**
image

**Before**
image

**After**
image

### CODE REVIEW

**Comment by Rodgers-M** [comment]
good work, the pages now look more presentable.

**Comment by MarloneA** [comment]
I like that you changed the social login buttons they where too big, good work

File: wger/core/templates/index.html [hunk 1]
```diff
@@ -199,6 +198,7 @@ <h4>{% trans "Weight"%}</h4>
         <a href="{% url 'weight:add' %}" class="btn btn-success btn-sm wger-modal-dialog">
             {% trans "Add weight entry" %}
         </a>
+        <a href="{% url 'core:fitbit-login' %}" class="btn btn-info btn-sm">Sync Fitbit</a>
```
Comment (MarloneA) [inline wger/core/templates/index.html]: th ksjdnfgkdjg . rhdfisdnfgidfsbgidfgdfkgdsf

**Review by paulrimiru — APPROVED** [review]
Looks good. I like the simplicity

**Pull request merged**